Test/Common/DomainTest/implementation/VolunteerOpportunityTest.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Ttu.Domain;
+
+namespace Ttu.DomainTest.implementation
+{
+    [TestClass]
+    public class VolunteerOpportunityTest
+    {
+        private VolunteerOpportunity _volunteerOpportunity;
+
+        private static readonly User TestUser = new User()
+        {
+            Description =  "description",
+            EmailAddress = "email",
+            FirstName = "First",
+            LastName = "Last",
+            Location = "here",
+            PasswordEncrypted = "Password",
+            RecordId = 500,
+            UserId = "8000"
+        };
+
+        private static readonly Organization TestOrganization = new Organization()
+        {
+            CreatedBy = TestUser,
+            Description = "MyOrg",
+            MissionStatement = "Don't be evil",
+            Name = "NotGoogle",
+            RecordId = 599,
+            Website = "www.nowhere.com"
+        };
+
+        [TestInitialize]
+        public void Setup()
```

Comment:
I like to call this method Arrange, as in the "AAA" (Arrange, Act, Assert) pattern for unit tests.